Missing script : bootstrap

Hello, @shuowu. First I want to thank you for this amazing browser extension for students. I am trying to add my own features to the extension but I am not familiar with Lerna. 

When I run `npm run bootstrap`, I see the following : 

```
npm ERR! missing script: bootstrap

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/Yomaru/.npm/_logs/2021-03-03T02_46_29_456Z-debug.log
```

There is no such script in package.json named `bootstrap`. I wonder how can I solve this.

--- a/README.md
+++ b/README.md
@@ -86,25 +86,13 @@ Suggested environment:
 
 ##### Install dependencies
 
-For NPM v6 or above:
-
 ```bash
-npm ci
-```
-
-For NPM before v6
-
-```bash
-npm install
+yarn install
 ```
 
 ##### Start development
 
-YiNote leverages [Lerna](https://github.com/lerna/lerna) to manage the monorepo.
-
-Please run `npm run bootstrap` to setup dependecies for each sub_modules before start.
-
-Then run `npm start` will start both `extension` and `playground` in development mode.
+Run `yarn start:ext`.
 
 For `content script` change, you need to reload the extension in `chrome://extensions` tab.
 

